internal/connect.go

```diff
@@ -24,18 +25,27 @@ import (
 	"github.com/hazelcast/hazelcast-go-client"
 )
 
+var InvalidStateErr = errors.New("invalid new state")
+
 type RESTCall struct {
 	url    string
 	params string
 }
 
 func CallClusterOperation(config *hazelcast.Config, operation string, state *string) (*string, error) {
-	obj := NewRESTCall(config, operation, state)
+	obj, err := NewRESTCall(config, operation, *state)
+	if err != nil {
+		if errors.Is(err, InvalidStateErr) {
+			fmt.Println("Error: invalid new state. It should be one the following:\n", ClusterStateActive, ClusterStateFrozen, ClusterStateNoMigration, ClusterStatePassive)
```

Comment:
`\n` won't work with `fmt.Println`.

Reply:
It looked okay to me but I can move it to the same line